vms/platformvm/validators/manager.go

```diff
@@ -29,7 +29,7 @@ import (
 const (
 	validatorSetsCacheSize        = 64
 	maxRecentlyAcceptedWindowSize = 256
```

Comment:
```suggestion
	maxRecentlyAcceptedWindowSize = 64
```
I think we also want to reduce the maximum window size as per the issue.